src/coverlet.msbuild.tasks/coverlet.msbuild.tasks.csproj

```diff
@@ -36,7 +36,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build.Utilities.Core" PrivateAssets="All" />
```

Comment:
sure that we won't pollute the task pacakge? can you compare the package?

Reply:
The new package contains less assemblies than V6.0.0 and content is identical with last nightly build.

<img width="1141" alt="image" src="https://github.com/coverlet-coverage/coverlet/assets/2315215/047ab314-98b6-466e-a07d-c2220a568234">

<img width="1139" alt="image" src="https://github.com/coverlet-coverage/coverlet/assets/2315215/2df66d31-83c9-40c8-b546-a435402ec48f">